meta/main.yml

```diff
@@ -10,9 +10,13 @@ galaxy_info:
       - all
 
 dependencies:
- - role: ANXS.python
+ - role: DavisRayM.python
    become: true
    become_user: "root"
    python_source_version: "{{ django_python_source_version }}"
+   python_packages:
```

Comment:
Where/how is this used?  Looks like its used in the python role?  Is there a way to make that clearer?